flink-dist/src/main/flink-bin/bin/standalone-job.sh

```diff
@@ -56,7 +56,7 @@ if [[ $STARTSTOP == "start" ]] || [[ $STARTSTOP == "start-foreground" ]]; then
     export FLINK_ENV_JAVA_OPTS="${FLINK_ENV_JAVA_OPTS} ${FLINK_ENV_JAVA_OPTS_JM}"
 
     # Startup parameters
-    ARGS+=("--configDir" "${FLINK_CONF_DIR}")
+    ARGS=("--configDir" "${FLINK_CONF_DIR}" "${ARGS[@]}")
```

Comment:
Why is this change necessary? `ARGS` should already contain the remaining command line arguments.

Reply:
You are right, this is not necessary. I thought the program arguments must be passed last, but that's obviously not true.

Reply:
No I think you're right and the additional command line arguments need to be passed last. Please remove the initial definition of `ARGS` if you fix it like this here.